src/utils/pathogenParameters.ts

```diff
@@ -0,0 +1,101 @@
+export const pathogenParameters: {
+  [key: string]: {
+    name: string;
+    genomeLength: number | "";
+    subsPerSitePerYear: number | "";
+    serialInterval: number | "";
+  };
+} = {
+  none: {
+    name: "",
+    genomeLength: "",
+    subsPerSitePerYear: "",
+    serialInterval: "",
+  },
+  sarscov2: {
+    name: "SARS-CoV-2",
+    genomeLength: 29903,
+    subsPerSitePerYear: 9.68e-4,
+    serialInterval: 5.2,
+  },
+  monkeypox: {
+    name: "Monkeypox",
+    genomeLength: 197209,
+    subsPerSitePerYear: 5.59e-5,
+    serialInterval: 9.8,
+  },
+  tb: {
+    name: "Tuberculosis",
+    genomeLength: 4411532,
+    subsPerSitePerYear: 1.33e-8,
+    serialInterval: 529,
+  },
+  measles: {
+    name: "Measles",
+    genomeLength: 15894,
+    subsPerSitePerYear: 4.97e-4,
+    serialInterval: 12,
+  },
+  mumps: {
+    name: "Mumps",
+    genomeLength: 15384,
+    subsPerSitePerYear: 4.35e-4,
+    serialInterval: 18,
+  },
+  ebola: {
+    name: "Ebola",
+    genomeLength: 18959,
+    subsPerSitePerYear: 1.05e-3,
+    serialInterval: 13,
+  },
+  other: {
+    name: "Other",
+    genomeLength: "",
+    subsPerSitePerYear: "",
+    serialInterval: "",
+  },
+};
+
+const factorial = (x: number) => {
+  if (x === 0 || x === 1) return 1;
+  for (var i = x - 1; i >= 1; i--) {
+    x *= i;
+  }
+  return x;
+};
+
+const poissonPMF = (subsPerTransmission: number, nMutations: number) => {
+  // estimate the probability of observing N mutations before a transmission, given the average number of substitutions per transmission
+  const lambda = subsPerTransmission;
+  const x = nMutations;
+
+  const numerator = Math.E ** (-1 * lambda) * lambda ** x + 0.000001;
```

Comment:
Nit: I'd like to see more parentheses -- reading even slightly complicated one liner math quickly makes me worried about order of operations. That said, I'm sure you're right and have tested it, so it's not really a big deal.

Reply:
I'm in the same boat. Prettier takes them out.